.coafile

```diff
@@ -1,7 +1,7 @@
 [DOCS]
 bears = SpaceConsistencyBear, LineLengthBear
 
-files = **/*.rst, README.md
+files = **/*.rst
```

Comment:
tbh this might be a problem with our fnmatch implementation rather than globs, I'll look into this today